src/api/store/userprofile.py

```diff
@@ -144,6 +144,8 @@ def _update_action_data_totals(action, household, delta):
       d.save()
 
 def can_be_deleted(user, context):
+  if context.user_is_super_admin and not user.is_super_admin:
+    return True
   if user.is_super_admin:
```

Comment:
This check can be moved up to eliminate the need to make the check again on line 147, to act as a guard clause